src/components/message/message.pw.tsx

```diff
@@ -0,0 +1,333 @@
+      page,
+    }) => {
+      await mount(<MessageComponent variant={variant} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check ${children} as children for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<Message>{children}</Message>);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check ${className} as className for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent className={className} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((id) => {
+    test(`should check ${id} as id for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent id={id} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when open is ${boolVal} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent open={boolVal} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  test(`should focus component when open is true and component has a ref for accessibility tests`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<MessageComponentWithRef />);
+    await checkAccessibility(page);
+  });
+
+  testData.forEach((title) => {
+    test(`should check ${title} as title for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent title={title} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  ([
+    [true, "rgba(0, 0, 0, 0)"],
+    [false, "rgb(255, 255, 255)"],
+  ] as [boolean, string][]).forEach(([boolVal, backgroundColor]) => {
+    test(`should check ${boolVal} for transparent background for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent transparent={boolVal} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  ([
+    [true, 50],
+    [false, 20],
+  ] as [boolean, number][]).forEach(([boolVal, paddingVal]) => {
+    test(`should check showCloseIcon when it's ${boolVal} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent showCloseIcon={boolVal} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((ariaLabel) => {
+    test(`should check closeButtonAriaLabel as ${ariaLabel} for accessibility tests`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<MessageComponent closeButtonAriaLabel={ariaLabel} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  test("should render message component for accessibility tests", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<MessageComponent />);
+
+    await checkAccessibility(page);
+  });
+});
```

Comment:
The original test has "onDismiss" prop checked accessibility in line 250-255 in "message.cy.tsx" file, could you add it in?